examples/index.html

```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
 
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
+      <div class="col-md-6">
+        <h2>Need Help?</h2>
+        <p>
+          Post a link to this and ask for help from other community members on <a href="https://publiclab.org/">PublicLab.org</a>
+        </p>
+        <p>
+          <a class="btn btn-primary" href="https://publiclab.org/questions">Ask a question &raquo;</a>
+        </p>
+      </div>
+      <div class="col-md-6">
```

Comment:
A class of col-sm-12 along with col-md-6 will be better

Reply:
Sure i will make the changes,😁

Reply:
Shall i change the link to github issues??